face/capture/src/main/java/com/simprints/face/capture/usecases/GetAndTurnOffPreparationInstructionsShowingUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.simprints.face.capture.usecases
+
+import androidx.annotation.VisibleForTesting
+import com.simprints.infra.security.SecurityManager
+import javax.inject.Inject
+import javax.inject.Singleton
+import androidx.core.content.edit
+
+@Singleton
+class GetAndTurnOffPreparationInstructionsShowingUseCase @Inject constructor(
```

Comment:
I would call it just `ShouldShowInstructionsScreenUseCase` to describe the purpose and leave the implementation details out of the name of the class.